src/App.js

```diff
@@ -1,7 +1,21 @@
 import React from "react";
+import "normalize.css";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import Navbar from "./components/NavBar";
+import GlobalStyle from "./globalStyles";
 
 function App() {
```

Comment:
For the App.js file, 
 
ToDo;

- Refactor app.js code to fit project size
- Create pages components to avoid conflicting interest
- Should Navbar be inside the Router component? React-router-dom insists